mesh/ncmesh.cpp

```diff
@@ -5601,6 +5625,22 @@ void NCMesh::PrintCoordinates(std::ostream &os) const
    }
 }
 
+void NCMesh::PrintVertexToKnot(std::ostream &os) const
+{
+   const int nv = vertex_to_knot.Size() / 4;
```

Comment:
My MFEM is a bit rusty, but I assume the `Array<int>` has a default size of 0, right?
If not then this statement might create issues, and `vertex_to_knot` would need to be set to zero explicitly somewhere (constructor).

I guess this is okay, just trying to be thorough...